constants/battle_anim_constants.asm

```diff
@@ -30,687 +30,687 @@ DEF BATTLEANIM_BASE_TILE EQU 7 * 7  ; Maximum size of a pokemon picture
-	const ANIM_OBJ_LEER
-	const ANIM_OBJ_LEER_TIP
-	const ANIM_OBJ_SCREEN
-	const ANIM_OBJ_HARDEN
-	const ANIM_OBJ_CHICK
-	const ANIM_OBJ_AMNESIA
-	const ANIM_OBJ_ASLEEP
-	const ANIM_OBJ_SKULL
-	const ANIM_OBJ_DIG_SAND
-	const ANIM_OBJ_DIG_PILE
-	const ANIM_OBJ_SAND
-	const ANIM_OBJ_PARALYZED
-	const ANIM_OBJ_STRING_SHOT
-	const ANIM_OBJ_HAZE
-	const ANIM_OBJ_MIST
-	const ANIM_OBJ_SMOG
-	const ANIM_OBJ_POISON_GAS
-	const ANIM_OBJ_HORN
-	const ANIM_OBJ_NEEDLE
-	const ANIM_OBJ_PETAL_DANCE
-	const ANIM_OBJ_SLUDGE_BOMB
-	const ANIM_OBJ_PAY_DAY
-	const ANIM_OBJ_SONICBOOM_JP_UNUSED
-	const ANIM_OBJ_MIMIC
-	const ANIM_OBJ_ATTRACT
-	const ANIM_OBJ_BONEMERANG
-	const ANIM_OBJ_BONE_CLUB
-	const ANIM_OBJ_BONE_RUSH
-	const ANIM_OBJ_SWIFT
-	const ANIM_OBJ_KINESIS
-	const ANIM_OBJ_FLASH
-	const ANIM_OBJ_SHINY
-	const ANIM_OBJ_SKY_ATTACK
-	const ANIM_OBJ_LICK
-	const ANIM_OBJ_WITHDRAW
-	const ANIM_OBJ_DRAIN
-	const ANIM_OBJ_GROWTH
-	const ANIM_OBJ_CONVERSION2
-	const ANIM_OBJ_SMOKE
-	const ANIM_OBJ_SMOKESCREEN
-	const ANIM_OBJ_SWORDS_DANCE
-	const ANIM_OBJ_SPEED_LINE
-	const ANIM_OBJ_SHARPEN
-	const ANIM_OBJ_DEFENSE_CURL
-	const ANIM_OBJ_METRONOME_HAND
-	const ANIM_OBJ_METRONOME_SPARKLE
-	const ANIM_OBJ_DISABLE
-	const ANIM_OBJ_AGILITY
-	const ANIM_OBJ_HEART
-	const ANIM_OBJ_FLAME_WHEEL
-	const ANIM_OBJ_SACRED_FIRE
-	const ANIM_OBJ_COTTON_SPORE
-	const ANIM_OBJ_MILK_DRINK
-	const ANIM_OBJ_ANGER
-	const ANIM_OBJ_HEAL_BELL
-	const ANIM_OBJ_HEAL_BELL_NOTE
-	const ANIM_OBJ_BATON_PASS
-	const ANIM_OBJ_LOCK_ON
-	const ANIM_OBJ_MIND_READER
-	const ANIM_OBJ_SAFEGUARD
-	const ANIM_OBJ_PROTECT
-	const ANIM_OBJ_THIEF
-	const ANIM_OBJ_OCTAZOOKA
-	const ANIM_OBJ_PRESENT
-	const ANIM_OBJ_SPIKES
-	const ANIM_OBJ_POWDER_SNOW
-	const ANIM_OBJ_DRAGONBREATH
-	const ANIM_OBJ_CONVERSION
-	const ANIM_OBJ_SPIDER_WEB
-	const ANIM_OBJ_CAKE_UNUSED
-	const ANIM_OBJ_NIGHTMARE
-	const ANIM_OBJ_IN_NIGHTMARE
-	const ANIM_OBJ_LOVELY_KISS
-	const ANIM_OBJ_SWEET_KISS
-	const ANIM_OBJ_SKETCH
-	const ANIM_OBJ_ENCORE_HAND
-	const ANIM_OBJ_ENCORE_STAR
-	const ANIM_OBJ_DESTINY_BOND
-	const ANIM_OBJ_MORNING_SUN
-	const ANIM_OBJ_GLIMMER
-	const ANIM_OBJ_MOONLIGHT
-	const ANIM_OBJ_HIDDEN_POWER
-	const ANIM_OBJ_CROSS_CHOP1
-	const ANIM_OBJ_CROSS_CHOP2
-	const ANIM_OBJ_SANDSTORM
-	const ANIM_OBJ_ZAP_CANNON
-	const ANIM_OBJ_SPITE
-	const ANIM_OBJ_CURSE
-	const ANIM_OBJ_PERISH_SONG
-	const ANIM_OBJ_FORESIGHT
-	const ANIM_OBJ_RAPID_SPIN
-	const ANIM_OBJ_SWAGGER
-	const ANIM_OBJ_BELLY_DRUM_HAND
-	const ANIM_OBJ_BELLY_DRUM_NOTE
-	const ANIM_OBJ_MEAN_LOOK
-	const ANIM_OBJ_BETA_PURSUIT
-	const ANIM_OBJ_SHOOTING_SPARKLE
-	const ANIM_OBJ_RAIN
-	const ANIM_OBJ_B0
-	const ANIM_OBJ_PSYCH_UP
-	const ANIM_OBJ_ANCIENTPOWER
-	const ANIM_OBJ_AEROBLAST
-	const ANIM_OBJ_SHADOW_BALL
-	const ANIM_OBJ_ROCK_SMASH
-	const ANIM_OBJ_FLOWER
-	const ANIM_OBJ_COTTON
-	const ANIM_OBJ_ENEMYFEET_1ROW
-	const ANIM_OBJ_PLAYERHEAD_1ROW
-	const ANIM_OBJ_ENEMYFEET_2ROW
-	const ANIM_OBJ_PLAYERHEAD_2ROW
+	const BATTLE_ANIM_INDEX_HIT_BIG_YFIX
```

Comment:
Let's have `; 00` comments, same as for the `SPRITE_ANIM_INDEX_*` constants. Likewise for the others in here.